notebooks/data-sources/oc-github-repo/github_PR_EDA.ipynb

```diff
@@ -7,7 +7,9 @@
    "source": [
```

Comment:
<p>Why is changed_file_type_vec a vector? It seems incongruous with the rest of the features which are all single items. Unless there is a reason I'm not seeing here, why not just make a feature be the count of changed files for each of the top file types? So we would have N new features instead of one that is a vector. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/293/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='78'/>

Reply:
<p>Thanks for pointing out the inconsistency. As we looked into this, we realize that by including the top N occurring features based on value counts, we might lose out on valuable information. At the same time, a large matrix (~1500 dims) may not provide the most value. </p><p> </p><p>Hence, we will compute correlation with time_to_merge for all the file_types and eventually keep only the top N new correlated features. So for this iteration, we can keep this a vector and also convert the other features like the words_in_title into a vector and eventually keep only the top correlated title_words. Does that make sense?</p>

Reply:
@oindrillac yes, sounds reasonable.

Reply:
@MichaelClifford added the changes 👍